### PR TITLE
[PXCT-17] add RTC RPC example

### PR DESCRIPTION
**Pull request by Hannes7eicher**
## What This PR Changes
This PR adds an example on how to access the RTC from the M4 core on the Arduino GIGA R1 WiFi.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by karlsoderby — CHANGES_REQUESTED** [review]
Hey @Hannes7eicher , appears to be working. However we are using the single sketch approach (uploading the same sketch to the different cores). 

I think it might be best to look at creating a specific M4 sketch, and a specific M7 sketch.

**Comment by Hannes7eicher** [comment]
@karlsoderby I think that's a good idea! I split apart the sketch. Take a look and let me know what you think.

File: content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-dual-core/giga-dual-core.md [hunk 1]
```diff
@@ -558,6 +558,90 @@ int servoMove(int angle) {
 }
 ```
 
+### RTC RPC
+
+This example demonstrates how the RTC can be accessed from the M4:
```
Comment (karlsoderby) [inline content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-dual-core/giga-dual-core.md:563]:
```suggestion
The Real-time Clock (RTC) can be accessed from the M4 core, and using RPC, we can read the values.

In this example, we set the RTC to a specific date (can be adjusted in the example), send it to the M7 via RPC, and finally prints it out in the Serial Monitor using the M7 core. 
```

File: content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-dual-core/giga-dual-core.md [hunk 2]
```diff
@@ -558,6 +558,90 @@ int servoMove(int angle) {
...
+#include <mbed_mktime.h>
+#include "RPC.h"
+
+constexpr unsigned long printInterval{ 1000 };
+unsigned long printNow{};
+
+void setup() {
+  if (RPC.begin()) {
+    RPC.println("M4: Reading the RTC.");
+    //RTCset()  //Uncomment if you need to set the RTC for the first time.
```
Comment (karlsoderby) [inline content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-dual-core/giga-dual-core.md:583]:
```suggestion
    RTCset(); //sets the RTC to start from a specific time & date
```

File: content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-dual-core/giga-dual-core.md [hunk 3]
```diff
@@ -558,6 +558,90 @@ int servoMove(int angle) {
...
+  t.tm_min = (58);           // 0-59
+  t.tm_hour = (11);          // 0-23
+  t.tm_mday = (1);           // 1-31
+  t.tm_mon = (9);            // 0-11  "0" = Jan, -1
+  t.tm_year = ((24) + 100);  // year since 1900,  current year + 100 + 1900 = correct year
+  set_time(mktime(&t));      // set RTC clock
+}
+```
+
+**M7 sketch:**
```
Comment (karlsoderby) [inline content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-dual-core/giga-dual-core.md:616]:
```suggestion
The M7 sketch is found below, and uses RPC to print out the incoming data from the M4.

**M7 sketch:**
```

**Review by karlsoderby — CHANGES_REQUESTED** [review]
Left some minor suggestion @Hannes7eicher . I think it's best to leave `RTC.set()` uncommented but let me know if there was a reason to leave like it that and we can adjust.

Otherwise everything looks great.

**Comment by Hannes7eicher** [comment]
All suggestions applied 😁

**Review by karlsoderby — APPROVED** [review]

**Pull request merged**